### PR TITLE
Replacing WebL10n with i18next in Blockrain

### PR DESCRIPTION
**Pull request by Vishalk91-4**
Fixes issue #1378 
All tutorials working in the languages mentioned in Blockrain activity

image

@llaske could you please review it

### CODE REVIEW

**Comment by Vishalk91-4** [comment]
@llaske, updated my code 
You can check it

File: activities/Blockrain.activity/lib/sugar-web/env.js
Comment (llaske) [inline activities/Blockrain.activity/lib/sugar-web/env.js:1]: This file is not supposed to be updated.

Comment (Vishalk91-4) [reply activities/Blockrain.activity/lib/sugar-web/env.js:1]: Yes, removed the spaces and changed back to original

**Comment by llaske** [comment]
Nice. Thanks

**Pull request merged**